android-kit-base/src/androidTest/kotlin/com/mparticle/kits/DataplanBlockingUserTests.kt

```diff
@@ -217,13 +223,15 @@ class DataplanBlockingUserTests : BaseKitOptionsTest() {
 
         datapoints[DataplanFilterImpl.USER_IDENTITIES_KEY] =
             allowedIdentities.keys.map { it.getEventsApiName() }.toHashSet()
-        AccessUtils.getKitManager().dataplanFilter = (DataplanFilterImpl(
-            datapoints,
-            Random.nextBoolean(),
-            Random.nextBoolean(),
-            Random.nextBoolean(),
-            true
-        ))
+        AccessUtils.getKitManager().dataplanFilter = (
+            DataplanFilterImpl(
+                datapoints,
+                Random.nextBoolean(),
+                Random.nextBoolean(),
+                Random.nextBoolean(),
+                true
+            )
+            )
```

Comment:
same nitpick, can you move this `(` back to align with AccessUtils?  Or did ktlint make you do it this way?